### PR TITLE
rebranding, dependencies update

### PR DESCRIPTION
**Pull request by KrzysztofKozlowskiOBSS**
image
image
image

### CODE REVIEW

File: markdown/1.x/en/documentation/communication.md [hunk 1]
```diff
@@ -50,10 +50,10 @@ now to be made to the shop system now. Change the URL settings in `ff-communicat
...
 
 
-#### 2. Forward the HTTP Request to FACT-Finder
+#### 2. Forward the HTTP Request to FactFinder
 
 ---
-The request made by FACT-Finder Web Components will look something like: 
+The request made by FactFinder Web Components will look something like: 
 
     `http://domain.com/FACT-Finder/Search.ff?query=shoe&channel=shoeshop_en`.
```
Comment (KrzysztofKozlowskiOBSS) [inline markdown/1.x/en/documentation/communication.md:58]: I intentionally skipped replacing `FACT-Finder` in the URLs. Please confirm if that's expected.

File: src/documentation-app.js [hunk 2]
```diff
@@ -36,7 +36,7 @@ class DocumentationApp extends ReduxMixin(PolymerElement) {
         --app-primary-color: #3c3c3b;;
         --app-secondary-color: white;
       
-        --button-active-background-color: #1b4385;
+        --button-active-background-color: #a175ea;
```
Comment (KrzysztofKozlowskiOBSS) [inline src/documentation-app.js:39]: Zrzut ekranu 2024-05-25 o 23 05 34
This color was not specified in the presentation, so I just used color picker to extract it

File: src/styles/sidebar-styles.js [hunk 3]
```diff
@@ -57,8 +57,8 @@ $_documentContainer.innerHTML = `<dom-module id="sidebar-styles">
             }
 
             .panel-menus > iron-selector a.iron-selected {
-                border-left: 6px solid rgb(27, 67, 133);
-                color: rgb(27, 67, 133);
+                border-left: 6px solid #230871;
+                color: #230871;
```
Comment (KrzysztofKozlowskiOBSS) [inline src/styles/sidebar-styles.js:61]: Zrzut ekranu 2024-05-25 o 23 09 48

these panels weren't mentioned in the design presentation, but I think they look better with the new color, too. Let me know if it should stay like that.

File: src/views/home-view.js [hunk 4]
```diff
@@ -90,7 +90,7 @@ class HomeView extends ViewMixin(ReduxMixin(PolymerElement)) {
     }
 
     button:hover {
-        background-color: #007bbf;
+        background-color: #230871;
```
Comment (KrzysztofKozlowskiOBSS) [inline src/views/home-view.js:93]: image
this button previously had a hover effect. I temporarily set the hover color to the same as regular. Let me know if we can turn off the hover effect completely. Or maybe this button should keep its old colors?

File: markdown/1.x/en/documentation/communication.md (hunk 1 above)
Comment (omikron-fp) [reply markdown/1.x/en/documentation/communication.md:58]: Yes, I think this was the right thing to do.

File: src/styles/sidebar-styles.js (hunk 3 above)
Comment (omikron-fp) [reply src/styles/sidebar-styles.js:61]: Looks good! Let's keep it.

File: src/views/home-view.js (hunk 4 above)
Comment (omikron-fp) [reply src/views/home-view.js:93]: Good call on this one! I think it's only consistent to also apply the new colours to this button.
We can do without the hover effect.

**Review by omikron-fp — APPROVED** [review]

File: src/views/home-view.js (hunk 4 above)
Comment (KrzysztofKozlowskiOBSS) [reply src/views/home-view.js]: hover effect removed

**Pull request merged**